src/modules/events/rc_loss_alarm.h

```diff
@@ -0,0 +1,83 @@
+/****************************************************************************
+ *
+ *   Copyright (c) 2018 PX4 Development Team. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ * 3. Neither the name PX4 nor the names of its contributors may be
+ *    used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+ * FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ * COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ * BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS
+ * OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED
+ * AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+ ****************************************************************************/
+
+/**
+ * @file rc_loss_alarm.h
+ * Tone alarm in the event of RC loss
+ *
+ */
+
+#pragma once
+
+#include "subscriber_handler.h"
+
+#include <uORB/uORB.h>
+#include <uORB/topics/vehicle_status.h>
+
+namespace events
+{
+namespace rc_loss
+{
+
+class RC_Loss_Alarm
+{
+public:
+
+	RC_Loss_Alarm(const events::SubscriberHandler &subscriber_handler);
+
+	/** regularily called to handle state updates */
+	void process();
+
+private:
+	/**
+	 * check for topic updates
+	 * @return true if one or more topics got updated
+	 */
+	bool check_for_updates();
+
+	/** Publish tune control to sound alarm */
+	void play_tune();
+
+	/** Publish tune control to stop any ongoing tune, including alarm */
+	void stop_tune();
```

Comment:
Oh, that's from the old implementation as a standalone module... But actually it's not even necessary since any ongoing tune can be stopped with `tune_control stop` in the px4 CLI.